pom.xml

```diff
@@ -19,7 +19,7 @@
     <properties>
         <revision>0.10</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.235.1</jenkins.version>
+        <jenkins.version>2.249.1</jenkins.version>
```

Comment:
This is only required to test the DurableStepTask https://github.com/v1v/workflow-durable-task-step-plugin/commit/adc9afa3d5761eb6aa358cf16e1e809610ecf5bd